packages/files-ui/src/Contexts/FilesContext.tsx

```diff
@@ -686,10 +734,16 @@ const FilesProvider = ({ children }: FilesContextProps) => {
   ) => {
     const UPLOAD_PATH = "/"
 
+    const cancelSource = axios.CancelToken.source()
+    const cancelToken = cancelSource.token
+
     const toastParams: ToastParams = {
       title: t`Sharing your file (Downloading)`,
       type: "success",
-      progress: 0
+      progress: 0,
+      toastPosition: "bottomRight",
+      onProgressCancel: cancelSource.cancel,
+      isClosable: false
```

Comment:
I think any toast should be closable, beside the ones from upload/download/sharing
```suggestion
      isClosable: true
```

Reply:
Yes, this one falls under sharing, this is the start of downloading part!